index.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Factory\AppFactory;
+use GuzzleHttp\Psr7\LazyOpenStream;
+
+require 'vendor/autoload.php';
+
+include "manifest.php";
+
+header("Content-Type: application/json");
+
+$app = AppFactory::create();
+
+$app->post('/function/{appUuid}', function (Request $request, Response $response, array $args) {
```

Comment:
What does the `function/{appUuid}` is made for ?

Reply:
To accept post requests on /function/{appUuid}, {appUuid} is a parameter of the url so we accept requests for any appUuid passed, we also accept /fucntion/test for dev-tool requests.

Reply:
Why does the dev-tool call the app with this path ? Can you look into the dev-tool please ?

Reply:
Should be replaced with a wildcard:

```suggestion
$app->post('/{path:.*}', function (Request $request, Response $response, array $args) {
```